fidelis-edr/operations.py

```diff
@@ -89,6 +89,8 @@ def get_authorisation_token(self):
             return response.get('data').get('token')
         else:
             logger.exception(str(response))
+            if 'Bad username or password' in str(response):
+                raise ConnectorError('Invalid endpoint or credentials')
```

Comment:
Why this change is needed?

Reply:
to make a consistent error message for most of the connectors. 
Bug - https://mantis.fortinet.com/bug_view_page.php?bug_id=0863405

Reply:
okey. got it.